src/config/field-overrides/programStage.js

```diff
@@ -25,8 +25,27 @@ const reportDateOptions = [
  *  or translateOptions through fieldOptions to the DropDown-component.
  */
 
-const ReportDateToUseDropDown = props =>
-    <DropDown {...props} options={reportDateOptions} translateOptions />;
+const ReportDateToUseDropDown = props => (
+    <DropDown {...props} options={reportDateOptions} translateOptions />
+);
+
+const NextScheduleDateField = props => {
+    const dataElementsOptions = props.model.programStageDataElements
+        .filter(psde => psde.dataElement.valueType === 'DATE')
+        .map(psde => ({
+            text: psde.dataElement.displayName,
+            value: psde.dataElement,
+        }));
```

Comment:
perhaps you could also use the `id` property and use that as `key` when you are mapping over the `menuItems`?

Reply:
That would be a breaking change to the dropdown. Options doesn't have an ID.

Reply:
You could of course check for an object if it has an id, and have a special case for this. However that seems unecessary, as the text should be a string and unique (two of the same thing in a dropdown shouldn't happen as that is very confusing and does not make sense).

Reply:
Thanks for looking into this. I would just leave things the way they are then.